cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -745,6 +1050,7 @@ void __host__ DecodePageDataFixed(cudf::detail::hostdevice_span<PageInfo> pages,
                                   size_t min_row,
                                   int level_type_size,
                                   bool has_nesting,
+                                  bool is_list,
```

Comment:
Yeah it's getting really long.  I'm doing that in the next PR (bools, which is otherwise small), and figured this PR is complicated enough as it is.